Sources/FuzzilliCli/main.swift

```diff
@@ -26,72 +26,74 @@ Usage:
-                                  This only includes programs that increase coverage.
-                                  This is useful for jump starting coverage for a wide range of JavaScript samples.
-                                  Can be used alongside importCorpusAll, and will run second.
-                                  Since all imported samples are asynchronously minimized, the corpus will show a smaller
-                                  than expected size until minimization completes.
-    --importCorpusMerge=path    : Imports a corpus of protobufs to start the initial fuzzing corpus.
-                                  This only keeps programs that increase coverage but does not attempt to minimize
-                                  the samples. This is mostly useful to merge existing corpora from previous fuzzing
-                                  sessions that will have redundant samples but which will already be minimized.
-    --networkMaster=host:port   : Run as master and accept connections from workers over the network. Note: it is
-                                  *highly* recommended to run network fuzzers in an isolated network!
-    --networkWorker=host:port   : Run as worker and connect to the specified master instance.
-    --noCorpusSynchronization   : Do not synchronize the corpus between instances. This way, the workers will behave
-                                  like separate instances (and may therefore stress different parts of the target),
-                                  but crashes will still be collected at one central location.
-    --dontFuzz                  : If used, this instace will not perform fuzzing. Can be useful for master instances.
-    --noAbstractInterpretation  : Disable abstract interpretation of FuzzIL programs during fuzzing. See
-                                  Configuration.swift for more details.
-    --collectRuntimeTypes       : Collect runtime type information for programs that are added to the corpus.
-    --diagnostics               : Enable saving of programs that failed or timed-out during execution. Also tracks
-                                  executions on the current REPRL instance.
-    --inspect=opt1,opt2,...     : Enable inspection options. The following options are available:
-                                      history: Additional .fuzzil.history files are written to disk for every program.
-                                               These describe in detail how the program was generated through mutations,
-                                               code generation, and minimization
-                                        types: Programs written to disk also contain variable type information as
-                                               determined by Fuzzilli as comments
-                                          all: All of the above
-""")
+    --profile=name                  : Select one of several preconfigured profiles.
```

Comment:
Oh, I added a tab to all options in a separate commit (https://github.com/googleprojectzero/fuzzilli/pull/370/commits/98a67bcabd451c5e73fb985abc02a5cf142dbed8), because `statisticsExportInterval` is just 1 character too long for keeping the `:` aligned with everything else. Let me know if that's okay with you :)

https://github.com/googleprojectzero/fuzzilli/blob/169fef7e1041816454e8689799485a863bac90d7/Sources/FuzzilliCli/main.swift#L59-L66

Reply:
Ah, hm we use spaces instead of tabs throughout the project, so I'd prefer if we can keep the spaces here :/ I guess that'd mean adding a space in front of every `:` here to fix the alignment

Reply:
Oh, got it! I deleted the tab I added and replaced it with a space

Reply:
perfect, thanks! :)